src/controllers/userController.ts

```diff
@@ -0,0 +1,27 @@
+import {UserRepository} from "../repositories/userRepository";
```

Comment:
```suggestion
import { UserRepository } from "../repositories/userRepository";
```

Y cuidado con los nombres de los ficheros.
UserController.ts
MovieController.ts
Si son clases yo pongo el nombre en mayúsculas (como en Java)